### PR TITLE
added comment about the die roll

### PR DESCRIPTION
**Pull request by jaxels20**

### CODE REVIEW

File: 1000_dice_rolls.R [hunk 1]
```diff
@@ -2,11 +2,15 @@ library(ggplot2)
 library(distr)
 set.seed(12345)
 
+
+
```
Comment (jaxels10) [inline 1000_dice_rolls.R]: Why are there these new lines?

File: 1000_dice_rolls.R [hunk 2]
```diff
@@ -2,11 +2,15 @@ library(ggplot2)
...
+#Simulating a die
 die <- 1:6
 roll <- function(n) {
   mean(sample(die, size = n, replace = TRUE))
 }
 
+
```
Comment (jaxels10) [inline 1000_dice_rolls.R]: Why did this newline get added?

**Review by jaxels10 — CHANGES_REQUESTED** [review]

File: 1000_dice_rolls.R (hunk 2 above)
Comment (jaxels20) [reply 1000_dice_rolls.R]: Har fjernet det

File: 1000_dice_rolls.R (hunk 1 above)
Comment (jaxels20) [reply 1000_dice_rolls.R]: Har fjernet det

**Review by jaxels10 — APPROVED** [review]

**Pull request merged**